### PR TITLE
Introduction and Conclusion

### PR DESCRIPTION
**Pull request by preetl**
as title xo

### CODE REVIEW

**Comment by ezgimez** [comment]
thanks girl

File: 18_conclusion/index.tex [hunk 1]
```diff
@@ -1 +1,18 @@
...
+    \begin{quote}
+        "We are each our own devil, and we make this world our hell."
+        \linebreak
+        \emph{Oscar Wilde}
+    \end{quote}
+\end{flushleft}
+
+Presented here, was the collective effort of 43 humble students united by a fascination with self-governance, Nash equilibria and a willingness to forgo their Christmas break. 
+
+This project successfully produced a simulation of a group of agents with the tools required to collaborate to solve collective action problems. Though limited in intelligence, in certain parameterisations, these agents were able to survive and thrive. This is no mean feat, especially considering that in the early days of this project, a frequently heard phrase was, 'how the fudge are we going to do this.'
```
Comment (ezgimez) [inline 18_conclusion/index.tex]:
```suggestion
This project successfully produced a simulation of a group of agents with the tools required to collaborate to solve collective action problems. Though limited in intelligence, in certain parameterisations, these agents were able to survive and thrive. This is no mean feat, especially considering that in the early days of this project, a frequently heard phrase was, 'how the fudge are we going to do this.'.
```

File: 00_introduction/index.tex [hunk 2]
```diff
@@ -1 +1,25 @@
...
+
+\begin{flushleft}
+\begin{quote}
+    "Therein is the tragedy. Each man is locked into a system that compels him to increase his herd without limit – in a world that is limited. Ruin is the destination toward which all men rush, each pursuing his own best interest in a society that believes in the freedom of the commons."
+    \linebreak
+    \emph{Ellinor Ostrom}
+\end{quote}
+\end{flushleft}
+
+In the current political climate, while shaking a fist angrily at the television screen, one often finds oneself asking the question, 'are human beings even capable of creating institutions for the common good?' As such, it is worth investigating whether electronic multi-agent systems are able to do a better than mere mortals, in the eventual hope of designing stronger, fairer socio-technical systems.
```
Comment (ezgimez) [inline 00_introduction/index.tex]:
```suggestion
In the current political climate, while shaking a fist angrily at the television screen, one often finds oneself asking the question, 'Are human beings even capable of creating institutions for the common good?'. As such, it is worth investigating whether electronic multi-agent systems are able to do a better than mere mortals, in the eventual hope of designing stronger, fairer socio-technical systems.
```

File: 00_introduction/index.tex [hunk 3]
```diff
@@ -1 +1,25 @@
...
+
+This project explores the ability of independent agents to self-organise in order to solve long and short-term collective risk dilemmas. The collective risk dilemma poses a challenge to a group of agents whereby they must balance their own interests against the interests of the collective. 
+
+The objective of this project was to design and implement two collective risk dilemmas as well as a system of agents capable of mitigating the risk. In order to do this successfully, the system contains adequate communication and self organisation tools for the agents to make use of.
```
Comment (ezgimez) [inline 00_introduction/index.tex]:
```suggestion
The objective of this project was to design and implement two collective risk dilemmas as well as a system of agents capable of mitigating the risk. In order to do this successfully, the system contains adequate communication and self-organisation tools for the agents to make use of.
```

File: 00_introduction/index.tex [hunk 4]
```diff
@@ -1 +1,25 @@
...
+
+The specific problem posed can be summarised as follows: islands in an archipelago are incentivised to pool their resources in order to collectively survive environmental disasters but must balance this against an interest in their own survival and properity. The method of resource replenishment is deer hunting and fishing. The populations of these resources are limited and neither of these foraging methods have guaranteed returns.
```
Comment (ezgimez) [inline 00_introduction/index.tex]:
```suggestion
The specific problem posed can be summarised as follows: Islands in an archipelago are incentivised to pool their resources in order to collectively survive environmental disasters but must balance this against an interest in their own survival and prosperity. The method of resource replenishment is deer hunting and fishing. The populations of these resources are limited and neither of these foraging methods has guaranteed returns. 
```

**Review by ezgimez — APPROVED** [review]
let's merge this, shall we

File: 00_introduction/index.tex [hunk 5]
```diff
@@ -1 +1,25 @@
-\chapter{Introduction}
\ No newline at end of file
+\chapter{Introduction}
+
+\begin{flushleft}
+\begin{quote}
+    "Therein is the tragedy. Each man is locked into a system that compels him to increase his herd without limit – in a world that is limited. Ruin is the destination toward which all men rush, each pursuing his own best interest in a society that believes in the freedom of the commons."
```
Comment (lhl2617) [inline 00_introduction/index.tex]:
```suggestion
    ``Therein is the tragedy. Each man is locked into a system that compels him to increase his herd without limit – in a world that is limited. Ruin is the destination toward which all men rush, each pursuing his own best interest in a society that believes in the freedom of the commons.''
```

File: 00_introduction/index.tex [hunk 6]
```diff
@@ -1 +1,25 @@
...
+    \linebreak
+    \emph{Ellinor Ostrom}
+\end{quote}
+\end{flushleft}
+
+In the current political climate, while shaking a fist angrily at the television screen, one often finds oneself asking the question, 'Are human beings even capable of creating institutions for the common good?'. As such, it is worth investigating whether electronic multi-agent systems are able to do a better than mere mortals, in the eventual hope of designing stronger, fairer socio-technical systems.
```
Comment (lhl2617) [inline 00_introduction/index.tex]:
```suggestion
In the current political climate, while shaking a fist angrily at the television screen, one often finds oneself asking the question: ``Are human beings even capable of creating institutions for the common good?''. As such, it is worth investigating whether electronic multi-agent systems are able to do a better than mere mortals, in the eventual hope of designing stronger, fairer socio-technical systems.
```

File: 18_conclusion/index.tex [hunk 7]
```diff
@@ -1 +1,18 @@
-\chapter{Conclusion}
\ No newline at end of file
+\chapter{Conclusion}
+
+
+\begin{flushleft}
+    \begin{quote}
+        "We are each our own devil, and we make this world our hell."
```
Comment (lhl2617) [inline 18_conclusion/index.tex]:
```suggestion
        ``We are each our own devil, and we make this world our hell.''
```

File: 18_conclusion/index.tex [hunk 8]
```diff
@@ -1 +1,18 @@
...
+        \linebreak
+        \emph{Oscar Wilde}
+    \end{quote}
+\end{flushleft}
+
+Presented here, was the collective effort of 43 humble students united by a fascination with self-governance, Nash equilibria and a willingness to forgo their Christmas break. 
+
+This project successfully produced a simulation of a group of agents with the tools required to collaborate to solve collective action problems. Though limited in intelligence, in certain parameterisations, these agents were able to survive and thrive. This is no mean feat, especially considering that in the early days of this project, a frequently heard phrase was, 'how the fudge are we going to do this.'.
```
Comment (lhl2617) [inline 18_conclusion/index.tex]:
```suggestion
This project successfully produced a simulation of a group of agents with the tools required to collaborate to solve collective action problems. Though limited in intelligence, in certain parameterisations, these agents were able to survive and thrive. This is no mean feat, especially considering that in the early days of this project, a frequently heard phrase was: ``how the fudge are we going to do this?''.
```

is `fudge` ok lmao

File: 18_conclusion/index.tex [hunk 9]
```diff
@@ -1 +1,18 @@
...
+
+Alongside the success of the simulation, we must also recognise the amazing accomplishment of meta self-organisation. The cohort spanned every timezone, personality, work ethic, haircut, political affiliation and ability to understand sarcasm (pardon the exaggerations here). Yet, despite all that divides us, we were able to come together and produce an impressive piece of work that we are particularly proud of. 
+
+I would like to conclude this report by thanking each and every one of my teammates for their outstanding contributions and unfailing committment to the delivery of this project. I think I speak for all of us when I say that I am amazed at what we have created and it has truly been an experience like no other, but let's not do it again. Please.
```
Comment (lhl2617) [inline 18_conclusion/index.tex]: lmaoo

File: 18_conclusion/index.tex [hunk 10]
```diff
@@ -1 +1,18 @@
...
+    \begin{quote}
+        ``We are each our own devil, and we make this world our hell.''
+        \linebreak
+        \emph{Oscar Wilde}
+    \end{quote}
+\end{flushleft}
+
+Presented here, was the collective effort of 43 humble students united by a fascination with self-governance, Nash equilibria and a willingness to forgo their Christmas break. 
+
+This project successfully produced a simulation of a group of agents with the tools required to collaborate to solve collective action problems. Though limited in intelligence, in certain parameterisations, these agents were able to survive and thrive. This is no mean feat, especially considering that in the early days of this project, a frequently heard phrase was: ``how the fudge are we going to do this?''.
```
Comment (ezgimez) [inline 18_conclusion/index.tex]:
```suggestion
This project successfully produced a simulation of a group of agents with the tools required to collaborate to solve collective action problems. Though limited in intelligence, in certain parameterisations, these agents were able to survive and thrive. This is no mean feat, especially considering that in the early days of this project, a frequently heard phrase was: ``How the fudge are we going to do this?''.
```

File: 18_conclusion/index.tex (hunk 9 above)
Comment (lhl2617) [reply 18_conclusion/index.tex]: who is `i` though -- is this ok?

**Review by lhl2617 — DISMISSED** [review]
remove `synctex(busy)` file + comments

File: 18_conclusion/index.tex (hunk 9 above)
Comment (ezgimez) [reply 18_conclusion/index.tex]: we cann say at tthe end

preet lalli

File: 18_conclusion/index.tex [hunk 11]
```diff
@@ -1 +1,18 @@
...
+
+This project successfully produced a simulation of a group of agents with the tools required to collaborate to solve collective action problems. Though limited in intelligence, in certain parameterisations, these agents were able to survive and thrive. This is no mean feat, especially considering that in the early days of this project, a frequently heard phrase was: ``How the fudge are we going to do this?''.
+
+Alongside the success of the simulation, we must also recognise the amazing accomplishment of meta self-organisation. The cohort spanned every timezone, personality, work ethic, haircut, political affiliation and ability to understand sarcasm (pardon the exaggerations here). Yet, despite all that divides us, we were able to come together and produce an impressive piece of work that we are particularly proud of. 
+
+I would like to conclude this report by thanking each and every one of my teammates for their outstanding contributions and unfailing committment to the delivery of this project. I think I speak for all of us when I say that I am amazed at what we have created and it has truly been an experience like no other, but let's not do it again. Please.
```
Comment (ezgimez) [inline 18_conclusion/index.tex]:
```suggestion
I would like to conclude this report by thanking each and every one of my teammates for their outstanding contributions and unfailing committment to the delivery of this project. I think I speak for all of us when I say that I am amazed at what we have created and it has truly been an experience like no other, but let's not do it again. Please. \\
- Preet Lalli
```

**Comment by Neelesh99** [comment]
> remove `synctex(busy)` file + comments

Syntex ded

File: 18_conclusion/index.tex [hunk 12]
```diff
@@ -1 +1,19 @@
...
+
+Alongside the success of the simulation, we must also recognise the amazing accomplishment of meta self-organisation. The cohort spanned every timezone, personality, work ethic, haircut, political affiliation and ability to understand sarcasm (pardon the exaggerations here). Yet, despite all that divides us, we were able to come together and produce an impressive piece of work that we are particularly proud of. 
+
+I would like to conclude this report by thanking each and every one of my teammates for their outstanding contributions and unfailing committment to the delivery of this project. I think I speak for all of us when I say that I am amazed at what we have created and it has truly been an experience like no other, but let's not do it again. Please. \\
+- Preet Lalli
```
Comment (Spruudis) [inline 18_conclusion/index.tex]:
```suggestion
We would like to conclude this report by thanking each and every one of our teammates for their outstanding contributions and unfailing commitment to the delivery of this project. We are all amazed at what we have created and it has truly been an experience like no other, but let's not do it again. Please. \\
```

Comment (Spruudis) [reply 18_conclusion/index.tex]: Seemed like Preet was thanking team 3, although that is not what she meant. I'd like to end by thanking everyone, cause honestly, you are all amazing

Comment (NGovani) [reply 18_conclusion/index.tex]: im gonna change teammates to classmates. I think its fine for it to be in first person with preets name at the end. She did take on the responsibility for writing this part. And it helps the rest of the humour for it be designated to one person

**Pull request merged**